File: src/moto-rev.hs
-- moto rev (jmcc)

import Sound.SC3 {- hsc3 -}

moto_rev :: UGen
moto_rev =
  let f = sinOsc KR 0.2 0 * 10 + 21
      s = lfPulse AR f (mce2 0 0.1) 0.1
  in clip2 (rlpf s 100 0.1) 0.4

main :: IO ()
main = audition (out 0 moto_rev)

File: gr/moto-rev.fs
\ moto rev (jmcc) #1

0.2 0 SinOsc.kr 10 * 21 + ( freq )
0 0.1 2 mce 0.1 LFPulse.ar ( osc )
100 0.1 RLPF 0.4 swap Clip2
play
